src/asciidoctor-core-api.js

```diff
@@ -5,32 +5,43 @@ var toHash = function (object) {
   return object;
 };
 
+var prepareOptions = function (options) {
```

Comment:
That is correct. It is now possible to pass the attributes as a Hash. We dig down to grab the value of the attributes key and coerce it if necessary. We're trying to shield the user from the low-level details here.

+1 to a test.